shared.js

```diff
@@ -144,6 +144,22 @@ function make_settings(callback, total_label) {
     });
 }
 
+function make_as_of(callback) {
+    return fetch(BASE_URL + "/info", {}).then(function(response) {
+        response.json().then(function(data) {
+            document.getElementById("as-of").innerHTML = "Updated as of: " + (new Date(data.as_of)).toLocaleString();
+
+            if (callback) {
```

Comment:
Hmm, that's true. I kept it from the other function, make_settings, but we don't use it. Will remove.